### DISCUSSION TITLE
Open Source Status

### DISCUSSION THREAD

**Discussion by pauzzz** [writer/writer-framework#456] [General]
Will this stay open source now that this has been acquired?

**Comment by ramedina86** [comment]
Yes, 100%. It'll stay open source under the same license, Apache 2.0.

**Comment by summa-code** [comment]
Any code that will remotely connect to your cloud without any consent? If we use this without using any AI features, how does that work. Would any part of the framework connect to your company's cloud for any reason even if we don't use these features?

**Reply by ramedina86** [reply]
There's currently no code that connects to the cloud, unless you explicitly:
- Use `writer deploy` to deploy the app
- Use any of Writer's AI services via `writer.ai`

There's no telemetry or anything of that kind built in. There are no plans to introduce it, either. However, though unlikely, that is subject to change.

The framework is fully usable without the AI features and we encourage people to use it even if they have to plan to use Writer's services.
